site/src/pages/_document.page.tsx

```diff
@@ -26,29 +26,30 @@ class MyDocument extends Document {
             content="A standardized way to create blocks whose contents are mapped to schemas, which are both
             human and machine-readable."
           />
-          <meta name="twitter:card" content="summary" />
+          <meta name="twitter:card" content="summary_large_image" />
           <meta name="twitter:title" content="Block Protocol" />
           <meta
             name="twitter:description"
             content="An open standard for data-driven blocks"
           />
-          <meta name="twitter:site" content="blockprotocol" />
+          <meta name="twitter:site" content="@blockprotocol" />
           <meta
             name="twitter:image:src"
             content="https://blockprotocol.org/assets/bp_twitter_cover.png"
           />
+
           <meta name="og:title" content="Block Protocol" />
           <meta
             name="og:description"
             content="An open standard for data-driven blocks"
           />
           <meta
-            name="og:image"
+            property="og:image"
             content="https://blockprotocol.org/assets/bp_og_cover.png"
           />
-          <meta name="og:url" content="https://blockprotocol.org/" />
```

Comment:
`og:url` is meant to be a normalized URL of a given page. Always referring to the home page URL can confuse social media platforms.

We can bring this tag back when we convert hardcoded meta tags into some kind of `<Meta title="..." />` component. It will need to be pulled out of `_document`. I can copy-paste this solution from a side project when it’s time to add custom previews for blocks etc.